viewer/reconstruction.html

```diff
@@ -1112,6 +1126,33 @@
 
             function onDocumentMouseDown(event) {
                 window.focus();
+
+                intersected = raycaster.intersectObjects(point_clouds);
+                if (
+                    intersected.length > 0 &&
+                    options.drawLineOnPointClick && 
+                    selectedCamera
+                ){
+                    var material = new THREE.LineBasicMaterial( { color: 0xFFC0CB } );
+                    var geometry = new THREE.Geometry();
+                    geometry.vertices.push(new THREE.Vector3( 
```

Comment:
This will use the center of the bounding sphere but it should use the optical center of the camera. You can use something like 
```
                    var shot = selectedCamera.reconstruction.shots[selectedCamera.shot_id];
                    var ocenter = opticalCenter(shot);
                    geometry.vertices.push(ocenter);
```
to use the optical center instead